EDDI/ChangeLog.xaml.cs

```diff
@@ -20,14 +20,15 @@ public ChangeLogWindow()
             try
             {
                 DirectoryInfo dir = new DirectoryInfo(System.IO.Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location));
-                markdown = Files.Read(dir + @"\ChangeLog.md");
+                markdown = File.ReadAllText(dir + @"\ChangeLog.md");
```

Comment:
This change is not necessary. Files.Read() utilizes File.ReadAllText().

Reply:
with Files.Read the result for character like é, ù, ç, à etc... changes, not with File.ReadAllText

Reply:
But with Files.Read the encoding is already defined as UTF-8. This change should not have any effect.
I've downloaded your branch and tested different characters (é, ù, ç, à)- they *do not* render correctly for me (without any changes to the code other than to add test characters in Markdown). I suspect that the browser used to render the text may make a difference?

Reply:
Before to post here i make a test with
markdown = Files.Read(dir + @"\Help.md");
blabla = File.ReadAllText(dir + @"\Help.md");
Logging.Info("markdown  ======>" + markdown);
Logging.Info("blabla ======>" + blabla );

and I paste the result somewhere in a post down